docs/user-guides/plugin/development.md

```diff
@@ -65,16 +65,16 @@ This will create a new folder `nginx-plugin` containing the _PluginDefinition_ i
 
 ## Modifying the PluginDefinition
 
-The generated _PluginDefinition_ contains a `plugin.yaml` file which defines the _PluginDefinition_. But there are still a few steps required to make it work.
+The generated _PluginDefinition_ contains a `plugindefinition.yaml` file which defines the _PluginDefinition_. But there are still a few steps required to make it work.
```

Comment:
the `greenhousectl` command actually creates a `pluginDefinition.yaml`
Is that a problem?